File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.cpp
class Solution {
public:
    int triangularSum(vector<int>& nums) {
        while(nums.size()>1){
            int i=0;
            while(i+1<nums.size()){
                nums[i]=(nums[i]+nums[i+1])%10;
                i+=1;
            }
            nums.erase(nums.begin()+nums.size()-1);
        }
        return nums[0];
    }
};

File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.py
class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        i=0
        while(len(nums)>1):
            i=0
            while(i<len(nums)-1):
                nums[i]=(nums[i]+nums[i+1])%10
                i+=1
            nums.pop(len(nums)-1)
        return nums[0]